Sources/StreamVideoSwiftUI/Utils/ClipCorners.swift

```diff
@@ -29,11 +31,14 @@ struct CornerClipper: ViewModifier {
             backgroundColor
                 .clipShape(RoundedCorners(radius: radius, corners: corners))
 
-            backgroundColorView
+            if extendToSafeArea {
```

Comment:
In cases where the view isn't expected to extend behind the safe area, we are hiding the backgroundColorView to avoid color-blend when using backgroundColors with opacity less than 1.